docs/source/deployment/lambda.md

```diff
@@ -30,7 +30,7 @@ First, install the `apollo-server-lambda` package:
 npm install apollo-server-lambda graphql
 ```
 
-Next, set up the schema's type definitions and resolvers, and pass them to the `ApolloServer` constructor like normal. Here, `ApolloServer` must be imported from `apollo-server-lambda`. It's also important to note that this file must be named `graphql.js`, as the config example in a later step depends on the filename.
+Next, set up the schema's type definitions and resolvers, and pass them to the `ApolloServer` constructor like normal. Here, `ApolloServer` must be imported from `apollo-server-lambda`. It's also important to note that this file must be named `graphql.js` and be located in the root of your project, as the config example in a later step depends on that filename and path.
```

Comment:
It actually doesn't need to be in the root, but that's a great call out to keep track of the file location!